equipment_handler/src/adcs_handler.c

```diff
@@ -2379,7 +2414,7 @@ ADCS_returnState ADCS_set_star_track_config(cubestar_config config) {
  */
 ADCS_returnState ADCS_get_cubesense_config(cubesense_config *config) {
 
-    uint8_t *telemetry = pvPortMalloc(113);
+    uint8_t *telemetry = (uint8_t*)pvPortMalloc(112);
```

Comment:
Should "112" be a macro?

Reply:
Should also be doing it for all handler commands. Magic numbers make things hard to read later